optimum/exporters/onnx/model_configs.py

```diff
@@ -393,15 +393,7 @@ class Phi3OnnxConfig(PhiOnnxConfig):
     ) + TextDecoderOnnxConfig.DUMMY_INPUT_GENERATOR_CLASSES
     DUMMY_PKV_GENERATOR_CLASS = MistralDummyPastKeyValuesGenerator
     NORMALIZED_CONFIG_CLASS = NormalizedTextConfigWithGQA
-    MIN_TRANSFORMERS_VERSION = version.parse("4.41.0")
-
-    def __init__(self, *args, **kwargs):
-        if is_transformers_version("==", "4.46.0"):
-            logger.error(
-                "Found transformers v4.46.0 while trying to exporting a Phi3 model, this specific version of transformers is not supported. "
```

Comment:
for 4.46 fixed in : https://github.com/huggingface/transformers/pull/34376
for 4.49 fixed in : https://github.com/huggingface/transformers/pull/36332
currently failing with v4.45 for num_beams > 1 : https://github.com/huggingface/optimum/actions/runs/14990250710

upgrading min transformers needed to v4.50 cc @IlyasMoutawwakil